ONSdigital/census-rm-ddl
# Alllow groundzero to be auto-regenerated

# Motivation and Context
Maintaining our groundzero scripts is manual and error-prone. Automating it should make it easier to see what DB changes have been made, and ensure code and DDL stay perfectly in sync.

# What has changed
Got rid of old Action Exporter repo. Added new Uac Qid Service repo. Scripted around other scripting enhancements. Replaced hand-written DDL with machine-generated DDL (based off current master branch).

# Checklist
Reminder: Make sure to version tag this after the PR is merged
* [ X ] Updated patch number
* [ X ] Updated version_tag

# How to test?
Make a change on a branch which would result in a DB/DDL change. Hack the `build_groundzero_ddl.sh` script to pull your branch instead of master. Check the change is present as expected in the ground zero DDL script.

# Links
Trello: https://trello.com/c/ybEYuI66
